troposphere/views/maintenance.py

```diff
@@ -17,6 +17,16 @@ def get_maintenance(request):
     return (records, disable_login, in_maintenance)
 
 
+def get_notices(request):
+    """
+    Returns a notice indicating details about a forthcoming maintenance period
+    """
+    records = MaintenanceNotice.active()
+    has_notice = records.count() > 0
+    record = records[0] if records.count() > 0 else records
+    return (has_notice, record)
```

Comment:
This will work for now but should be cleaned up in a future refactor of MaintenanceNotice